src/Symfony/Bundle/TwigBundle/Resources/config/twig.xml

```diff
@@ -52,6 +52,7 @@
         <service id="twig.loader.filesystem" class="Symfony\Bundle\TwigBundle\Loader\FilesystemLoader" public="false">
             <argument type="service" id="templating.locator" />
             <argument type="service" id="templating.name_parser" />
+            <argument>%kernel.root_dir%/..</argument>
```

Comment:
this should use the same value than for ``twig.loader.native_filesystem`` instead, which is configured in a compiler pass and deals with passing the path without using ``/..`` (which would not play well with systems not supporting realpath)